pkg/kubernetes/utils.go

```diff
@@ -18,10 +18,10 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
-var mDir string = "../../deployments"
-
 func getDirPath(dirName string) string {
 	currentPath, err := os.Getwd()
+	mDir := os.Getenv("KUBERNETES_MANIFEST_DIR")
```

Comment:
Oh, Thank you! But we have to passing the env from main package when create Kubernetes object. 😢

Reply:
Please add the filed for manifest path in here (https://github.com/tokamak-network/tokamak-titond-backend/blob/818b847db2bf70aabae1cd724c212dbf1085165f/pkg/kubernetes/k8s.go#L9).
And pass it here (https://github.com/tokamak-network/tokamak-titond-backend/blob/818b847db2bf70aabae1cd724c212dbf1085165f/cmd/titond/main.go#L37).

Reply:
i got it! i added manifestPath config to kubernetes struct. thanks!